src/main/kotlin/no/nav/syfo/personstatus/infrastructure/database/repository/PersonOversiktStatusRepository.kt

```diff
@@ -269,7 +269,7 @@ class PersonOversiktStatusRepository(private val database: DatabaseInterface) :
 
     override fun searchPerson(searchQuery: SearchQuery): List<PersonOversiktStatus> {
         val initials = searchQuery.initials.value.toList()
-        val baseQuery = "SELECT * FROM PERSON_OVERSIKT_STATUS p WHERE "
+        val baseQuery = "SELECT * FROM PERSON_OVERSIKT_STATUS p WHERE oppfolgingstilfelle_end + INTERVAL '16 DAY' >= now() AND "
```

Comment:
Et alternativ er å bare søke på navn (og etter hvert fødselsdato) i query her og så filtrere resultatet på aktivt oppfølgingstilfelle i koden. Usikker på hva som er best ytelsesmessig, hva tenker dere?

Reply:
Ytelsesmessig så er det nok bedre å gjøre det som en del av query, men er ikke akkurat sikkert det har så stor betydning ettersom det kommer an på antall. Når det er sagt så tenker jeg på generell basis at det ikke alltid er like hensiktsmessig å gjøre alt i en query hvis gitt query blir ulesbar (eller vanskelig å vedlikeholde), men nå høres ikke det ut som tilfelle her.

Har dere noen måter dere kan utføre lasttester på her? Teknisk sett så kunne man vel laget tester på dette, men usikker på hvor godt det reflekterer faktisk ytelse (sikkert en god indikator da men).

Reply:
Ja, har også en følelse av at det er bedre å gjøre det som en del av query her og den er såpass enkel her at det ikke er noe problem for lesbarheten. 
Vi har ingen rigg for lasttester her så jeg tenker dette bare er noe vi må prøve ut i prod (kan bruke unleash og teste med begrenset antall brukere til å begynne med) for å se hvordan faktisk ytelse på søket blir.